guides/common/modules/con_installing-and-configuring-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,8 @@
+[id="installing-and-configuring-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment"]
+= Installing and configuring the {Team} Insights On-Premises advisor engine
+
+The {Team} Insights On-Premises (IoP) advisor engine helps {RHEL} systems perform efficiently, keeping them secure and available.
+It analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings. 
+When you deploy the {Team} IoP advisor engine locally on {Project}, you can generate Insights recommendations without sending system data to {Team} services.
+
```

Comment:
@bangelic The Jira ticket says it's a Technology Preview, right? Then we need to add the following disclaimer:

```suggestion

:FeatureName: IoP advisor engine
include::snip_technology-preview.adoc[]

```
See also: [snip_technology-preview.adoc](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/snip_technology-preview.adoc?plain=1)
Feel free to rephrase the `FeatureName` value.